lib/rubocop/cop/mixin/allowed_methods.rb

```diff
@@ -3,7 +3,8 @@
 module RuboCop
   module Cop
     # This module encapsulates the ability to allow certain methods when
-    # parsing.
+    # parsing. Even if the code is in offense, if it contains methods
+    # that are allowed, it will deal with not be registered as an offense.
```

Comment:
"deal with" seems redundant here. 

You should probably also mentioned that this is functionality equivalent to the `IgnoredMethods` class that is deprecated and will be removed in RuboCop 2.0.

Reply:
Thank you so much. I updated this PR.